classes/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parents_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parents_permission = parents_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_service?
+    of_age? || @parents_permission
+  end
```

Comment:
- [ ] [OPTIONAL] Great job making the requested changes. Kindly note that this specific method should not be private. When you put any methods after the private keyword all the methods below it are made private. Kindly refer to the [previous reviewer's comment](https://github.com/AsimKhan2019/School_Library/pull/1#discussion_r980161509) I strongly advise you to make this change.

Reply:
@Nemwel-Boniface Thank you for the feedback. I realized my mistake and updated the code accordingly and placed the method before private.